.env.example

```diff
@@ -17,9 +17,13 @@ DATABASE_URL="file:./db.sqlite"
 # You can generate a new secret on the command line with:
 # openssl rand -base64 32
 # https://next-auth.js.org/configuration/options#secret
-# NEXTAUTH_SECRET=""
 NEXTAUTH_URL="http://localhost:3000"
+NEXTAUTH_SECRET="43Mj+s+kkl41yOLoeLx6qAJbiFvssEpIOkdVooO8Rac="
 
 # Next Auth Discord Provider
-DISCORD_CLIENT_ID=""
-DISCORD_CLIENT_SECRET=""
+DISCORD_CLIENT_ID="43Mj+s+kkl41yOLoeLx6qAJbiFvssEpIOkdVooO8Rac=j"
```

Comment:
This looks like you didn't branch from latest main, can you remove the discord ones?

Reply:
If it fails without these then never mind, will fix that in separate PR

Reply:
`src/env.mjs` was requiring it. I have removed the discord relevant lines.